src/CategoryItem.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Created by Katarzyna_Bak on 27.03.2017.
+ */
+
+import React, { Component } from 'react';
+import './CategoryItem.css';
+
+
+function Subcategories(props) {
```

Comment:
Is it possible to extract it to separate file and reuse in `Categories` component? There is almost the same code (see `categoryItems` const).